applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/images/mcuboot/prj_fast_pair.conf

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_SIZE_OPTIMIZATIONS=y
+
+# Disable memory guard to avoid false faults in application after boot
+CONFIG_HW_STACK_PROTECTION=n
+
+CONFIG_SYSTEM_CLOCK_NO_WAIT=y
+
+CONFIG_MAIN_STACK_SIZE=10240
+CONFIG_MBEDTLS_CFG_FILE="mcuboot-mbedtls-cfg.h"
+
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+CONFIG_BOOT_BOOTSTRAP=n
+
+CONFIG_BOOT_VERSION_CMP_USE_BUILD_NUMBER=y
+
+# Flash
+CONFIG_FLASH=y
```

Comment:
seems that this comment is a bit unecessary as it only relates to one option which is named the same

Reply:
Removed globally